book/src/frontends/README.md

```diff
@@ -0,0 +1,3 @@
+# Frontends
+
+While any front end can be implemented in powdr asm, powdr comes built-in with several front-ends for popular instruction set architectures.
```

Comment:
```suggestion
While any front end can be implemented in powdr asm, powdr comes built-in with several frontends for popular instruction set architectures.
```
Haha should also converge on how to spell frontend. In this paragraph alone we have "frontends", "front end" and "front-ends"

Reply:
single word it is